draft-bartle-tls-deprecate-ffdhe.md

```diff
@@ -217,9 +227,13 @@ already considered bad practice since they do not provide forward secrecy. Howev
 Raccoon revealed that timing side channels in processing TLS premaster secrets may be
 exploited to reveal the encrypted premaster secret.
 
-Raccoon does not apply to non-ephemeral elliptic curve DH suites, since the same timing
-side channel does not exist. However, such re-use is still discouraged and thus deprecated
-in this document.
+For non-ephemeral elliptic curve DH cipher suites, invalid curve attacks
+similarly exploit side channels to extract the secret from a long-lived public
+key. These attacks have been shown to be practical against real-world TLS
+implementations {{ICA}}.
+
+Therefore, DH public key re-use, over any algebraic group, is discouraged and
+thus deprecated in this document.
```

Comment:
If we're not going to say people "MUST NOT" use ECDH, we can't use the word "deprecate," can we?

Reply:
Right. Maybe this?
Therefore, DH public key re-use is discouraged in this document, and deprecated in cases where the security risk cannot be reasonably mitigated.

Reply:
I think we need to be more specific. We've already stated in the first paragraph of this section that FFDH is prohibited, so maybe just say "Therefore, this document discourages the reuse of elliptic curve DH public keys."